cupyx/scipy/sparse/compressed.py

```diff
@@ -336,3 +559,154 @@ def sum_duplicates(self):
         coo.sum_duplicates()
         self.__init__(coo.asformat(self.format))
         self._has_canonical_format = True
+
+    #####################
+    # Reduce operations #
+    #####################
+
+    def _minor_reduce(self, ufunc, axis, nonzero):
+        """Reduce nonzeros with a ufunc over the minor axis when non-empty
+        Can be applied to a function of self.data by supplying data parameter.
+        Warning: this does not call sum_duplicates()
+
```

Comment:
Please write docstrings that match the format
https://github.com/cupy/cupy/blob/master/cupyx/linalg/solve.py#L11-L21

Reply:
This should be corrected.

Reply:
Sorry to be annoying with this, but can you change the Returns to have only the datatype instead of the name+type?

Reply:
No worries. I'll know the format for next time. :) It should be fixed in the latest version.